docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -4,14 +4,36 @@ description: Define tasks that can be run pre- or post-deployment
 weight: 89
 ---
 
-
 A `KeptnTaskDefinition` defines tasks
 that are run by the Keptn Lifecycle Toolkit
 as part of the pre- and post-deployment phases of a
 [KeptnApp](./app.md) or
 [KeptnWorkload](../concepts/workloads/).
 
-## Yaml Synopsis
+A Keptn task runs as an application
+[container](https://kubernetes.io/docs/concepts/containers/),
+which runs as part of a Keptn
+[job](https://kubernetes.io/docs/concepts/workloads/controllers/job/).
+A Keptn job is part of a task.
+
+TODO: That last sentence doesn't make sense --
+perhaps "task" needs a modifier.
+
+A Keptn task can be defined in either of two ways:
+
+* Define a Kubernetes application container,
```

Comment:
This paragraph sounds very complicated, can we simplify it?  Something like:
A Keptn Task can be defined in a keptnTaskDefinition, either specifying a default runtime (JS deno - python ) or any user-defined container.

Reply:
This is rewritten now and reflects the three options with links to the synopsis and examples for each.  I think it is a bit clearer